varname/utils.py

```diff
@@ -277,6 +279,50 @@ def get_argument_sources(node: ast.Call,
     bound_args = signature.bind_partial(*arg_sources, **kwarg_sources)
     return bound_args.arguments
 
+def get_function_called_argname(
+        frame: FrameType,
+        node: ast.AST
+) -> Callable:
+    """Get the function who called argname"""
+    # We need node to be ast.Call
+    if not isinstance(node, ast.Call):
+        raise VarnameRetrievingError(
+            f"Expect an 'ast.Call' node, but got {type(node)!r}. "
+            "Are you using 'argname' inside a function?"
+        )
+
+    # variable
+    if isinstance(node.func, ast.Name):
+        return frame.f_locals.get(node.func.id,
+                                  frame.f_globals.get(node.func.id))
```

Comment:
I can't actually think how it would happen but you should probably raise an error here if the variable is not found rather than returning None.